### DISCUSSION TITLE
How to update 2 files with the same dependecy but in unique branches?

### DISCUSSION THREAD

**Discussion by toast-gear** [renovatebot/renovate#12900] [Request Help]
I have the below Renovate config, it works well:

```
{
  "extends": ["config:base"],
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchPackageNames": ["actions/runner"],
      "extractVersion": "^v(?<version>.*)$",
      "automerge": true
    }
  ],
  "regexManagers": [
    {
      "fileMatch": [
        ".github/workflows/ubuntu-18-04-runners.yml",
        ],
      "matchStrings": ["RUNNER_VERSION: +(?<currentValue>.*?)\\n"],
      "depNameTemplate": "actions/runner",
      "datasourceTemplate": "github-releases"
    }
  ]
}
```

This config gets Renovate to update `ubuntu-18-04-runners.yml` to newer version. What I want next is to get the config to update a second file `ubuntu-20-04-runners.yml` with the same change but in a separate branch and as a seperate PR.

I'm really not sure how to change Renovate, I got the impression from the docs this isn't possible?

**Comment by rarkins** [comment]
I don't fully understand your example, but if you want separate PRs then you need different branch names. The quickest way to do that is usually to define a value for `additionalBranchPrefix` for one or both of the branches which you want to separate.

**Reply by toast-gear** [reply]
I could do this but isn't that going to raise a single PR which includes changes to both files as they're are targeting the same dependency (the GitHub release)?

```
{
  "extends": ["config:base"],
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchPackageNames": ["actions/runner"],
      "extractVersion": "^v(?<version>.*)$",
      "automerge": true
    }
  ],
  "regexManagers": [
    {
      "fileMatch": [
        ".github/workflows/ubuntu-18-04-runners.yml",
        ],
      "matchStrings": ["RUNNER_VERSION: +(?<currentValue>.*?)\\n"],
      "depNameTemplate": "actions/runner",
      "datasourceTemplate": "github-releases"
    },
    {
      "fileMatch": [
        ".github/workflows/ubuntu-20-04-runners.yml",
        ],
      "matchStrings": ["RUNNER_VERSION: +(?<currentValue>.*?)\\n"],
      "depNameTemplate": "actions/runner",
      "datasourceTemplate": "github-releases"
    }
  ]
}
```

**Reply by rarkins** [reply]
Look at the documentation for `packageRules`. You can use e.g. `matchCurrentVersion` or `matchFiles`
